sql/server.go

```diff
@@ -192,3 +195,52 @@ func rollbackTxnAndReturnResultWithError(planMaker *planner, err error) driver.R
 	errProto.SetResponseGoError(err)
 	return driver.Result{Error: &errProto}
 }
+
+// GetParameters returns the Params slice as a `parameters`.
+func GetParameters(r driver.Request) Parameters {
```

Comment:
let's remove and inline this function. It kind of made sense as a method but now it's totally useless.